src/EnergyPlus/DataSurfaces.hh

```diff
@@ -911,6 +911,7 @@ namespace DataSurfaces {
         // Air boundaries
         int SolarEnclIndex;     // Pointer to solar enclosure this surface belongs to
         int SolarEnclSurfIndex; //  Pointer to solar enclosure surface data, ZoneSolarInfo(n).SurfacePtr(RadEnclSurfIndex) points to this surface
+        bool IsAirBoundarySurf; // True if surface is an air boundary surface (Construction:AirBoundary),
```

Comment:
Add new property to `Surface`, because most (all?) AFN unit tests do not have Construction data available.